src/compiler-src.md

```diff
@@ -101,12 +93,9 @@ At the very bottom of the dependency tree are a handful of crates that are used
 by the whole compiler (e.g. [`rustc_span`]). The very early parts of the
 compilation process (e.g. parsing and the AST) depend on only these.
 
-Pretty soon after the AST is constructed, the compiler's [query system][query]
-gets set up.  The query system is set up in a clever way using function
-pointers. This allows us to break dependencies between crates, allowing more
-parallel compilation.
-
-However, since the query system is defined in [`rustc_middle`], nearly all
+After the AST is constructed, the compiler's [query system][query]
```

Comment:
Some things happen between AST construction and query system setup I think.

Reply:
Can we address missing content in a follow-up?

Reply:
Sure, but I'd rather you revert the wording change since it now sounds like it's asserting that nothing happens in between.

Reply:
@pierwill can you address this comment?

Reply:
Maybe 

```suggestion
After the AST is constructed and other early analysis is done, the compiler's [query system][query]
```

Reply:
Seems good enough to me 👍